transports/transport.h

```diff
@@ -221,6 +221,11 @@ struct janus_transport {
 	 * @param[in] session_id The session ID that was closed (if the transport cares)
 	 * @param[in] timeout Whether the cause for the session closure is a timeout (this may interest transport plugins more) */
 	void (* const session_over)(janus_transport_session *transport, guint64 session_id, gboolean timeout);
+	/*! \brief Method to notify the transport plugin that a session it owned was claimed by another transport
+	 * \note A transport plugin should close the connection as a result of such an event
+	 * @param[in] transport Pointer to the transport session instance
+	 * @param[in] session_id The session ID that was closed (if the transport cares) */
+	void (* const session_claimed)(janus_transport_session *transport, guint64 session_id);
```

Comment:
I think `session_claimed` should have more information. Right now, this only tells the transport a session has been claimed. Anyway, it doesn't tell the transport whether the session is still handled by the same transport or not, and if so which one. I'm not exactly sure what we should add here so no need to do anything right now (I need to think about this), just wanted to raise the point to have this discussion.